cmd/clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -90,7 +90,7 @@ func (d *ClusterDeployer) Create(cluster *clusterv1.Cluster, machines []*cluster
 	}
 
 	klog.Infof("Creating control plane %v in namespace %q", controlPlaneMachines[0].Name, cluster.Namespace)
-	if err := phases.ApplyMachines(bootstrapClient, cluster.Namespace, []*clusterv1.Machine{controlPlaneMachines[0]}); err != nil {
+	if err := phases.ApplyMachines(bootstrapClient, cluster.Namespace, []*clusterv1.Machine{controlPlaneMachines[0]}, machineDeployments); err != nil {
```

Comment:
Replace `machineDeployments ` with `[]*clusterv1.MachineDeployment{}` because:
- It contains both `controlplane` machines and `node` machines.
- HA Control Planes cannot be deployed using `machineDeployments` (who joins who?)

Reply:
Yep, you're totally right.

Reply:
Fixed. We'll definitely have to look further into pivoting MachineDeployments, but I don't think it's super critical now.